### PR TITLE
Tweak LDAPS troubleshooting docs

### PR DESCRIPTION
**Pull request by zmb3**
Make it clear that an LDAP connection reset can be due to attempting to connect on port 389 (which is what our original instructions advised)

### CODE REVIEW

**Review by russjones — APPROVED** [review]
Approved, merge once @klizhentas has approved.

File: docs/pages/desktop-access/troubleshooting.mdx
Comment (klizhentas) [inline docs/pages/desktop-access/troubleshooting.mdx:167]: you can highlight port numbers by back-ticks

**Review by klizhentas — APPROVED** [review]

**Comment by russjones** [comment]
Will buddy in https://github.com/gravitational/teleport/pull/9323.

**Pull request closed without merging**
